inspirehep/modules/workflows/tasks/refextract.py

```diff
@@ -154,7 +154,7 @@ def extract_references_from_pdf_url(url, custom_kbs_file, source=None):
         headers=refextract_request_headers,
         data=json.dumps({"url": url, "journal_kb_data": custom_kbs_file})
     )
-    if response.status_code != 200 and response.json.get("messsage"):
+    if response.status_code != 200 and response.json().get("messsage"):
         LOGGER.info("Couldn't extract references from url!")
```

Comment:
I think we should fail the workflow in this case, rather than silently not extract refs.